src/site/content/en/blog/audio-output-latency/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Synchronize audio and video playback on the web
+subhead: >
+  A new Web Audio feature makes it possible to properly achieve AV-sync.
+date: 2022-05-25
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/9nYDoT45VHvNAtTfLciY.jpeg
+alt: A photo of a video editing timeline in Adobe Premiere Pro
+authors: 
+  - beaufortfrancois
+description: >
+  A new Web Audio feature makes it possible to properly achieve AV-sync.
+tags:
+  - blog
+  - audio
+  - media
+---
+
+The `outputLatency` property of an `AudioContext` instance provides an estimate of audio hardware's output latency (for example, that of Bluetooth earbuds or of an external USB audio interface). This property is necessary to coordinate/compensate the latency from the input to the output. And it is also very useful when you need to synchronize a video stream and an audio stream produced by the [Web Audio API](https://developer.mozilla.org/docs/Web/API/Web_Audio_API). The process of synchronizing video and audio streams together is called [AV synchronization](https://en.wikipedia.org/wiki/Audio-to-video_synchronization).
```

Comment:
> This property is necessary to coordinate/compensate the latency from the input to the output.

Can we clarify this? The output latency can be useful when you want to:
- Synchronize the existing audio material and the newly recorded material. (in a music production scenario)
- Synchronize the Web Audio output and other media (e.g. video or MIDI playback).

So "coordinating from the input to the output" doesn't seem clear enough on what it does. Perhaps I am missing something.